ItemsListApp/test/ItemsListApp.Api.UnitTests/Tests/Controllers/ItemsControllerUnitTests.cs

```diff
@@ -58,16 +58,15 @@ public void SetUp()
 
             var action = await _controller.GetAsync();
             var response = await action.ExecuteAsync(CancellationToken.None);
-
             IEnumerable<Item> actual;
             response.TryGetContentValue(out actual);
 
             Assert.That(response.IsSuccessStatusCode);
-            Assert.That(actual, Is.EquivalentTo(expected));
+            Assert.That(actual, Is.EqualTo(expected).AsCollection.UsingItemComparer());
         }
 
         [Test]
-        public async Task TestPost()
+        public async Task Post_Text_ReturnsCreatedItem()
```

Comment:
Consider, again for `task 2`, using `ValidText` in the test name – again, it is more descriptive and leads to a desire for at least one more test (once some validation takes places in your controller).